docs/guidelines/content/player-compatibility-testing.md

```diff
@@ -9,12 +9,12 @@ The biggest barrier to adding additional hash support to sets is the time requir
 ## Process Overview
 Players interested in conducting new hash testing should contact the set maintainer via site direct message and request to conduct compatibility testing. A set maintainer is any active developer who is an author on a set. If all set authors are inactive, [QATeam](https://retroachievements.org/user/QATeam) is the set maintainer.
 
-If the set maintainer agrees to the compatibility testing, they will link the test hash. It will be the tester's responsibility to complete the tests required by the set maintainer.
+If the set maintainer agrees to the compatibility testing, they will link the test hash, marked it "Untested" and assigner the requestor as "Tester" so the test may commence. It will be the tester's responsibility to complete the tests required by the set maintainer.
 
-If approved for a compatibility test, the requested hash will be linked with a **COMPATIBILITY TEST - NOT FOR PLAYER USE** label. The tester will be required to master or complete the set within a time frame determined by the set maintainer. The tester should create save states before as many achievements as possible. The tester will report issues directly to the set maintainer via site message and shall not open tickets if using the test hash. When the tester masters or completes the achievement set, the test hash will be renamed to its appropriate title. Its patch will then be added to RAPatches. The hash will be considered supported.
+The tester will be required to master or complete the set within a time frame determined by the set maintainer. The tester should create save states before as many achievements as possible. The tester will report issues directly to the set maintainer via site message and shall not open tickets if using the test hash. When the tester masters or completes the achievement set, the test hash will be renamed to its appropriate title. Its patch will then be added to RAPatches. The hash will be considered supported.
```

Comment:
"achievements as possible." - > "achievements as possible, especially any achievements that fail to trigger"
"When the tester " -> "If and only when the tester"